src/WormholeReceiver.sol

```diff
@@ -22,6 +22,10 @@ abstract contract WormholeReceiver is Ownable, WormholeBase {
   /// @dev A mapping of message hash to a boolean indicating delivery.
   mapping(bytes32 => bool) public seenDeliveryVaaHashes;
 
+  constructor(address owner) Ownable() {
```

Comment:
```suggestion
  constructor(address owner) Ownable(owner) {
```

nit: owner settable in `Ownable` constructor which may be more idiomatic than calling `transferOwnership` here

Reply:
I thought so as well but when I try to do this I get the below error

```